packages/core/test/lib/utils/traceData.test.ts

```diff
@@ -115,15 +103,15 @@ describe('getTracingMetaTags', () => {
             traceFlags: TRACE_FLAG_SAMPLED,
           };
         },
-        transaction: undefined,
       },
       mockedScope,
       undefined,
     );
 
     expect(tags).toEqual({
```

Comment:
l: due to above's suggestion

```suggestion
    expect(traceData).toEqual({
```